### PR TITLE
Feature/vost 84 saving has seen tutorial

### PR DESCRIPTION
**Pull request by Vanethos**
- When the user has seen the tutorial, then don't show it again
- If the user clicks on "Tutorial" in the about screen, he will be taken to the tutorial

### CODE REVIEW

File: lib/presentation/app.dart [hunk 1]
```diff
@@ -32,9 +32,10 @@ class VostApp extends StatelessWidget {
               title: 'Vost App',
               bloc: DependencyProvider.of(context).getHomeBloc(),
             ),
-        routeSplash: (context) => SplashPage(title: 'Splash Page'),
+        routeSplash: (context) => SplashPage(title: 'Splash Page', bloc: DependencyProvider.of(context).getSplashBloc(),),
```
Comment (miguelpruivo) [inline lib/presentation/app.dart]: Shouldn't this be on internalization support?

**Comment by Vanethos** [comment]
Resolved all the conflicts

File: lib/presentation/app.dart (hunk 1 above)
Comment (Vanethos) [reply lib/presentation/app.dart]: Same as the other issue - this will be done when all the copy is given

**Review by miguelpruivo — APPROVED** [review]
Internalization will be added on the future.

**Pull request merged**